code/obj/item/gun/kinetic.dm

```diff
@@ -9,7 +9,7 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 	var/caliber = null // Can be a list too. The .357 Mag revolver can also chamber .38 Spc rounds, for instance (Convair880).
 	var/has_empty_state = 0 //does this gun have a special icon state for having no ammo lefT?
 	var/gildable = 0 //can this gun be affected by the [Helios] medal reward?
-
+	var/gilded = FALSE //Currently used for handling gilded shotguns, if we ever want to gild the Colt SAA, its here
```

Comment:
doc comment please

Reply:
I think I did what you wanted not entirely sure